src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -75,7 +92,18 @@ public Long addCaseToSuit(Long suitId, CaseDTO caseDTO) {
         return caze.getId();
     }
 
+    public Long addCaseToSuit(Long suitId, EditCaseDTO editCaseDTO) {
+        CaseDTO caseDTO = new CaseDTO(editCaseDTO.getId(),editCaseDTO.getName(), editCaseDTO.getDescription(), new ArrayList<>(),
+            editCaseDTO.getPriority(), new HashSet<>(), editCaseDTO.getStatus());
+
+        return addCaseToSuit(suitId, caseDTO);
+    }
+
     public void updateCase(Long suitId, Long caseId, EditCaseDTO editCaseDTO) {
+        CaseDTO caseDTO = new CaseDTO(editCaseDTO.getId(), editCaseDTO.getDescription(),
```

Comment:
EditCaseDTO contains Action field that does not use in this method but can cause json parse exception in updateCase controller.